ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -956,8 +987,8 @@
 		83CBB9F71A601CBA00E9B192 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 0810;
-				LastUpgradeCheck = 0910;
+				LastSwiftUpdateCheck = 810;
+				LastUpgradeCheck = 910;
```

Comment:
Xcode and react-native fight about whether to add the leading-0 here. Xcode adds it, so if you could revert this change that'd be great.